src/lib/button-toggle/button-toggle.ts

```diff
@@ -223,6 +223,15 @@ export class MdButtonToggleGroup extends _MdButtonToggleGroupMixinBase
    */
   setDisabledState(isDisabled: boolean): void {
     this.disabled = isDisabled;
+    this._markButtonTogglesForCheck();
+  }
+
+  private _markButtonTogglesForCheck() {
+    if (this._buttonToggles) {
+      this._buttonToggles.forEach((toggle) => {
```

Comment:
Can be turned into a one-liner: `this._buttonToggles.forEach(toggle => toggle._markForCheck());`.

Reply:
Thanks for review!